src/main/java/com/stripe/model/Mandate.java

```diff
@@ -184,6 +198,9 @@ public static Mandate retrieve(
     @SerializedName("au_becs_debit")
     AuBecsDebit auBecsDebit;
 
+    @SerializedName("bacs_debit")
```

Comment:
This should be _inside_ `PaymentMethodDetails` but... the others are not, so I'll just ignore and we'll fix in a major.